server/src/creators.js

```diff
@@ -20,6 +20,9 @@ export const getCreators = (models) => {
       if (typeof startTime === 'undefined' || typeof endTime === 'undefined') {
         return Promise.reject(Error('Must pass start and end times'));
       }
+      if (endTime < startTime) {
+        return Promise.reject(Error('endTime must be greater than startTime'));
```

Comment:
you should probably specify:
`endTime must be greater than or equal to startTime`
or change the if statement to use `<=`.

Reply:
Err, dont you mean i would have to specify AND change the statement.

> 100 <= 100
true
> 100 < 100
false

thus, endTime < startTime means "endTime must be greater than startTime" an equal endTime and startTime would be true and pass this test, thus my error message is correct assuming we are happy to allow equal times.